packages/material-ui-icons/package.json

```
@@ -25,7 +25,7 @@
     "build": "yarn build:cjs && yarn build:esm && yarn build:es && yarn build:typings && yarn build:copy-files",
     "build:cjs": "cross-env NODE_ENV=production BABEL_ENV=cjs babel --config-file ../../babel.config.js ./src --out-dir ./build --ignore \"**/*.test.js\"",
     "build:esm": "cross-env NODE_ENV=production BABEL_ENV=esm babel --config-file ../../babel.config.js ./src --out-dir ./build/esm --ignore \"**/*.test.js\"",
-    "build:es": "cross-env NODE_ENV=production BABEL_ENV=es babel --config-file ../../babel.config.js ./src --out-dir ./build/es --ignore \"**/*.test.js\"",
+    "build:es": "echo 'skip es folder' & exit 0",
```

```suggestion
    "build:es": "echo 'skip es folder'",
```
`&& exit 0` is a noop. 
- If the first command returned 0 then we return `exit 0` => same exit code
- If the first command return non-zero we won't reach `exit 0` => same exit code